packages/keyboard/android/build.gradle

```
@@ -1,11 +1,44 @@
+import groovy.json.JsonSlurper
+
+def findNodeModulesWithRN(path = '..', level = 0) {
+    def rnPackageJsonFile = new File(rootProject.projectDir, path + "/node_modules/react-native/package.json")
+    if (file(rnPackageJsonFile).exists()) {
+        return new File(rootProject.projectDir, path + "/node_modules").absolutePath
+    }
+    // If it more that 10 than we assume that it's not an RN project
+    if (level == 10) {
+        throw "You can't use uikit.keyboard in not RN project"
+    }
+    return findNodeModulesWithRN("${path}/..", level + 1)
+}
+def NODE_MODULES_DIR = findNodeModulesWithRN();
+
+def FOR_HERMES = System.getenv("FOR_HERMES") == "True"
+rootProject.getSubprojects().forEach({project ->
+    if (project.plugins.hasPlugin("com.android.application")) {
+        FOR_HERMES = project.ext.react.enableHermes
+    }
+})
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:4.2.2'
+    }
+}
+
 apply plugin: 'com.android.library'
 
 project.ext {
-    buildToolsVersion = rootProject.ext.has("buildToolsVersion") ? rootProject.ext.buildToolsVersion : '27.0.3'
-    minSdkVersion = rootProject.ext.has("minSdkVersion") ? rootProject.ext.minSdkVersion : 19
-    compileSdkVersion = rootProject.ext.has("compileSdkVersion") ? rootProject.ext.compileSdkVersion : 27
-    targetSdkVersion = rootProject.ext.has("targetSdkVersion") ? rootProject.ext.targetSdkVersion : 25
-    supportLibVersion = rootProject.ext.has("supportLibVersion") ? rootProject.ext.supportLibVersion  : '27.1.1'
+    buildToolsVersion = rootProject.ext.has("buildToolsVersion") ? rootProject.ext.buildToolsVersion : '30.0.2'
+    minSdkVersion = rootProject.ext.has("minSdkVersion") ? rootProject.ext.minSdkVersion : 21
```

You're right. We forgot about this place

```suggestion
    minSdkVersion = rootProject.ext.has("minSdkVersion") ? rootProject.ext.minSdkVersion : 24
```